python/alb_ssl_policy.py

```diff
@@ -0,0 +1,432 @@
+#
+# This file made available under CC0 1.0 Universal (https://creativecommons.org/publicdomain/zero/1.0/legalcode)
+#
+# Created with the Rule Development Kit: https://github.com/awslabs/aws-config-rdk
+# Can be used stand-alone or with the Rule Compliance Engine: https://github.com/awslabs/aws-config-engine-for-compliance-as-code
+#
+'''
+#####################################
+##           Gherkin               ##
+#####################################
+Rule Name:
+  alb_ssl_policy_check
+Description:
+  Checks that listener of Application Load Balancer has configured applicable SSL policy.
+Trigger:
+  Configuration change on AWS::LoadBalancingV2::LoadBalancer.
+Resource Type to report on:
+  AWS::LoadBalancingV2::LoadBalancer
+Rule Parameters:
+  | ---------------------- | --------- | -------------------------------------------------------------------------------------- |
+  | Parameter Name         | Type      | Description                                                                            |
+  | ---------------------- | --------- | -------------------------------------------------------------------------------------- |
+  | ValidPolicies          | Optional  | The list of ssl policy name which check whether lister configred with comma separated  |
+  | ---------------------- | --------- | -------------------------------------------------------------------------------------- |
+
+Feature:
+  In order to: apply ssl policy to listers
+  As: a Security Officer
+  I want: to chase configuration of listener ssl policy of an application load balancer.

```

Comment:
I want: to verify that the configuration of listener ssl policy of an application load balancer is correct.